test/opselect_constants_int_scalar.cl

```diff
@@ -0,0 +1,72 @@
+// RUN: clspv  %s -S -o %t.spvasm
```

Comment:
I recently had to update ~900 tests when making an unrelated change. Could you please cut the number of CHECK statements to make this a more targeted test? I would like to get to a state where fewer tests  need to be updated for unrelated changes. It also makes the tests easier to follow.

In this case I think the following would be sufficient:
```
CHECK: [[uint:%[a-zA-Z0-9_]+]] = OpTypeInt 32 0
CHECK-DAG: [[zero:%[a-zA-Z0-9_]+]] = OpConstant [[uint]] 0
CHECK-DAG: [[one:%[[a-zA-Z0-9_]+]] = OpConstant [[uint]] 1
CHECK: OpSelect [[uint]] {{.*}} [[zero]] [[one]]
```

Similarly other tests could be reduced in terms of number of checks.

Reply:
I agree. I'm having to change a lot of tests as part of another change I have in progress and this is particularly painful and a waste of our time.